tests/integration/test_requirements_dev.py

```
@@ -0,0 +1,547 @@
+        
+        Parses REQUIREMENTS_FILE, counts comparison operators (`>=`, `==`, `<=`, `>`, `<`, `~=`) found in non-empty version specifications, and asserts that at least 50% of all detected operators are `>=`. On failure, raises an assertion that includes the percentage of `>=` usages and the operator counts.
+        """
+        assert REQUIREMENTS_FILE.exists(), "requirements-dev.txt not found"
+        
+        requirements = parse_requirements(REQUIREMENTS_FILE)
+        
+        # Count operator usage using regex pattern that matches operators followed by version
+        # This avoids substring matching issues (e.g., >= being counted as both >= and >)
+        operator_counts = {'>=': 0, '==': 0, '<=': 0, '>': 0, '<': 0, '~=': 0}
+        
+        for pkg, version_spec in requirements:
```

Ruff (0.14.8)

Loop control variable `pkg` not used within loop body

Rename unused `pkg` to `_pkg`

(B007)